proxy/envoy/testdata/rds-ingress-weighted.json.golden

```diff
@@ -0,0 +1,28 @@
+{
+  "virtual_hosts": [
+   {
+    "name": "world.com",
+    "domains": [
+     "world.com"
+    ],
+    "routes": [
+     {
+      "path": "/hello",
+      "host_rewrite": "world.default.svc.cluster.local:80",
+      "weighted_clusters": {
+       "clusters": [
+        {
+         "name": "out.c76febe0f151b2f8abe0f377d2052c0fbbfb959d",
+         "weight": 75
+        },
+        {
+         "name": "out.66fcc955b8875b19844f9eaf6cfda47c778c609e",
+         "weight": 25
+        }
```

Comment:
If there is a route rule on destination that uses headers, there should be two routes - one with header match, and another for catch-all. Both should be added, as long as they are compatible.